entries/z/zybooks.com.json

```
@@ -0,0 +1,13 @@
+{
+  "zyBooks": {
+    "domain": "zybooks.com",
+    "url": "https://www.zybooks.com",
+    "contact": {
+      "facebook": "zyBooks",
+      "twitter": "zyBooks"
+    },
+    "keywords": [
+      "education"
+    ]
```

The target audience seems to be US and CA residents thus a regions array would be appropriate here.
```suggestion
    ],
    "regions": [
      "ca",
      "us"
    ]
```